packages/tools/src/utilities/segmentation/utilities.ts

```diff
@@ -10,7 +10,7 @@ export type ThresholdInformation = {
   upper: number;
 };
 
-export default function getBrushToolInstances(toolGroupId) {
+export default function getBrushToolInstances(toolGroupId, toolName) {
```

Comment:
I think there are other calls that need to include this new parameter. I believe the unit tests are also failing because of this.